controlpanel/frontend/jinja2/home.html

```diff
@@ -13,6 +13,7 @@ <h2 class="govuk-heading-m">Superuser functions</h2>
     <li><a href="{{ url('list-users') }}">List all users</a></li>
     <li><a href="{{ url('list-all-apps') }}">List all apps</a></li>
     <li><a href="{{ url('list-all-datasources') }}">List all data sources</a></li>
+    <li><a href="{{ url('list-releases') }}">List all releases</a></li>
```

Comment:
Similar comment to one above about making this more explicit:

`url('list-tools-releases')` and `List all tools releases` (or similar) avoid potential confusion for "non-initiated" people which may confuse this with webapp releases somehow.

Reply:
Fixed.